specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-04-01-preview/apimapis.json

```diff
@@ -2143,7 +2143,7 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/apis/{apiId}/schemas": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/apis/{apiId}/apischemas": {
```

Comment:
I don't see we have changed the `ApiSchemaController`

Reply:
@solankisamir , feel free not to block merging 2021-04-01-preview into main on this PR. We'll request it separately.

Reply:
@solankisamir It's going back to "schemas"